index.html

```diff
@@ -0,0 +1,62 @@
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Cuaca - The Minimalistic Weather App</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.css.map">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.min.css">
+        <link href="https://fonts.googleapis.com/css?family=Ubuntu:300,400,500,700" rel="stylesheet">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+        <link rel="stylesheet" href="css/custom.css">
+        <script defer src="https://use.fontawesome.com/releases/v5.1.0/js/all.js"></script>
+    </head>
+    <body class="site">
+        <section class="hero is-light">
+                <div class="hero-body">
+                    <div class="container">
+                        <h1 class="title">
+                            <i class="fas fa-umbrella-beach"></i> Cuaca
+                        </h1>
+                        <h2 class="subtitle">
+                            Where you can find the information about the weather here!
+                        </h2>
+                    </div>
+                </div>
+        </section>
+        <br>
+        <main class="site-content">
+            <section>
+                <div class="container" id="contain">
+                    <div class="row center">
+                        <h1 class="lesserTitle">Enter City:</h1>
+                    </div>
+                    <br>
+                    <div class="field">
+                        <div class="control">
+                            <input class="input is-rounded" id="input" type="text" placeholder="Enter City Here">
+                        </div>
+                    </div>
+                </div>
+            </section>
+        </main>
+        <footer class="footer is-primary">
+            <div class="content has-text-centered">
+              <p> Powered by <a href="https://openweathermap.org"><strong>OpenWeather</strong></a></p>
+            </div>
+            <div class="content has-text-centered">
+                <a href="https://icons8.com">Icon pack by <strong>Icons8</strong></a>
+            </div>
+            <div class="content has-text-centered">
+                <i class="fab fa-github" href="https://github.com/wakatobi/cuaca"></i>
+            </div>
+        </footer>
+        <script src="js/weather.js" async defer></script>
```

Comment:
I think `async` and `defer` here will not make sense unless the `script` tag is placed on the `head`.